tests/test_graph_components/providers/test_domain_without_responses_provider.py

```diff
@@ -0,0 +1,60 @@
+from typing import Text
+
+import pytest
+from rasa.graph_components.providers.domain_without_response_provider import (
+    DomainWithoutResponseProvider,
+)
+from rasa.engine.graph import ExecutionContext
+from rasa.engine.storage.resource import Resource
+from rasa.engine.storage.storage import ModelStorage
+from rasa.shared.core.domain import KEY_E2E_ACTIONS, KEY_RESPONSES, Domain
+
+
+@pytest.mark.parametrize(
+    "domain_yml",
+    [
+        ("data/test_domains/travel_form.yml"),
+        ("data/test_domains/mixed_retrieval_intents.yml"),
+    ],
+)
+def test_copy_without_responses(
+    default_model_storage: ModelStorage,
+    default_execution_context: ExecutionContext,
+    domain_yml: Text,
+):
+
+    component = DomainWithoutResponseProvider.create(
+        {"arbitrary-unused": 234},
+        default_model_storage,
+        Resource("xy"),
+        default_execution_context,
+    )
+
+    domain = Domain.from_file(path=domain_yml,)
```

Comment:
For better readability I would rename this to `original_domain` and replace `given` occurences with `original`.

Reply:
also done